ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_dhcp_option_space.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Infoblox, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+from email.policy import default
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: b1_dhcp_option_space
+author: "Akhilesh Kabade (@akhilesh-kabade-infoblox), Sriram Kannan(@kannans)"
+short_description: Configure DHCP Option Space on Infoblox BloxOne DDI
+version_added: "1.0.1"
+description:
+  -  Create, Update and Delete Option spaces on Infoblox BloxOne DDI. This module manages the IPAM Optionspace object using BloxOne REST APIs.
+requirements:
+  - requests
+options:
+  api_key:
+    description:
+      - Configures the API token for authentication against Infoblox BloxOne patform.
+    type: str
+    required: true
+  host:
+    description:
+      - Configures the Infoblox BloxOne host URL.
+    type: dict
+    required: true
+  name:
+    description:
+      - Configures the name of object to fetch, add, update or remove from the system. User can also update the name as it is possible
+        to pass a dict containing I(new_name), I(old_name).
+    type: str
+    required: true
+  protocol:
+    description:
+      - Configures the protocol,this field is mandatory. It should be either ip4 or ip6.
+    type: str
+  tags:
+    description:
+      - Configures the tags associated with the object to add or update from the system.
+    type: list
+  comment:
+    description:
+      - Configures the comment/description for the object to add or update from the system.
+    type: str
+  state:
+    description:
+      - Configures the state of the object on BloxOne DDI. When this value is set to C(get), the object
+        details are fetched (if present) from the platform, when this value is set to C(present), the object
+        is configured on the platform and when this value is set to C(absent)
+        the value is removed (if necessary) from the platform.
+    type: str
+    default: present
+    choices:
+      - present
+      - absent
+    required: true
+'''
+
+EXAMPLES = '''
+   - name: Create Option Space
+      b1_dhcp_option_space:
```

Comment:
addition space is present in this section. Can you correct for all 
```suggestion
     b1_dhcp_option_space:
```